server/service/src/stocktake/update.rs

```diff
@@ -214,10 +218,22 @@ fn generate_stock_line_update(
     } else {
         None
     };
+
+    let location_movement = if counted_number_of_packs <= 0.0 {
```

Comment:
can I request a change please?
this should query for existing location movements which match `store_id` and `stock_line_id` and have `None` for `exit_datetime`
If one is found, add a value for the `exit_datetime` ..if none is found then no action required. We only need to record the exit if there was an entry value.

Reply:
[7e0e1ba](https://github.com/openmsupply/open-msupply/pull/812/commits/7e0e1ba5fae4a081654e493717fdea5b400f99d2)